Content.Client/Administration/AdminNameOverlay.cs

```diff
@@ -57,16 +61,24 @@ protected override void Draw(in OverlayDrawArgs args)
                 continue;
             }
 
-            var lineoffset = new Vector2(0f, 11f);
+            var uiScale = _configurationManager.GetCVar(CVars.DisplayUIScale);
+            if (uiScale == 0.0f)
+            {
+                // DisplayUIScale can be 0 if left at the default
+                uiScale = _userInterfaceManager.DefaultUIScale;
+            }
```

Comment:
Get the UI scale from `IUserInterfaceManager.RootControl` instead, that way it properly integrates with all parameters. The UI scale selection logic is too complex to copy into content.